src/browser_action.css

```diff
@@ -25,16 +50,28 @@ header {
   height: 48px;
   background: #0086f4;
   padding: 12px;
-  position: fixed;
+  position: relative;
   width: 100%;
   top: 0;
   left: 0;
   right: 0;
   z-index: 10;
 }
 
-.header-placeholder {
-  height: 48px;
+.section-container {
+  display: flex;
+  overflow: hidden;
+  height: 432px;
+}
+
+section {
+  -webkit-box-flex: 1;
```

Comment:
Existing usage of 'flex' (e.g. for '.start-time' below) does not have any vendor-prefixed variants. We should be consistent. Do you know when Chrome started supporting flex?

Reply:
> We should be consistent.

You are right, I added the other cases by habit.

From Chrome 21 to 28 with prefix `-webkit-` and since 29 with just using `flex`.

Reply:
We should generally assume people are using Chrome evergreen (i.e. the latest version). The kinds of users who hunt down and install extensions like this usually keep their browser updated. Let’s drop the prefixes.